modules/odc-grouping-multiple-components-within-an-application.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// applications/application-life-cycle-management/odc-viewing-application-composition-using-the-topology-view.adoc
+
+[id="odc-grouping-multiple-components-within-an-application_{context}"]
+= Grouping multiple components within an application
+
+You can use the *Add* page to add multiple components or services to your project and use the *Topology* page to group applications and resources within an application group.
+For this tutorial, add a MongoDB database service to an existing application with a Node.js component as follows:
+
+
+.Prerequisites
+
+* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+//<link>
+
+.Procedure
+
+. Create and deploy the MongoDB service to the `myproject` project as follows:
+.. In the Developer Perspective, navigate to the *+Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
+.. Click on the *MongoDB* option to see the details for the service.
+.. Click *Instantiate Template* to see an auto populated template with details for the MongoDb service and click *Create* to create the service.
+
+. On the left navigation panel, click *Topology* to see the MongoDB service deployed in your project.
+. To add the MongoDB service to the existing application group, select the *mongodb* Pod and drag it to the application, the MongoDB service is added to the existing application group.
+. Dragging a component and adding it to an application group automatically adds the required labels to the component. Click on the MongoDB service node to see the label `app.kubernetes.io/part-of=myapp` added to the *Labels* section in the *Overview* Panel.
++
+image::odc_app_grouping_label.png[Application Grouping]
+
+Alternatively, you can also add the component to an application as follows:
+
+. To add the MongoDB service to your application, click on the *mongodb* Pod to see the *Overview* panel to the right.
+. Click the *Actions* drop-down on the upper-right of the panel and select *Edit Application Grouping*.
```

Comment:
click the *Select an Application* drop-down menu, and select 

(drop-down is an adjective according to IBM style)